src/pages-and-resources/discussions/app-list/AppCard.jsx

```diff
@@ -41,6 +41,7 @@ function AppCard({
           aria-label={intl.formatMessage(messages.selectApp, {
             appName: intl.formatMessage(messages[`appName-${app.id}`]),
           })}
+          data-testid="checkbox"
```

Comment:
=> `data-testid="app-checkbox"`

Reply:
You can use queryByLabelText here, rather than queryByTestId, which would be preferable.